scripts/version

```diff
@@ -12,8 +12,10 @@ GIT_TAG=${DRONE_TAG:-$(git tag -l --contains HEAD | head -n 1)}
 LAST_TAG=${GIT_TAG:-'v0.0.0'}
 
 if [[ -z "$DIRTY" && -n "$GIT_TAG" ]]; then
+    export DASHBOARD_VERSION=$GIT_TAG
```

Comment:
Thought about this for a while. I think we should rename or splash comments everywhere. It needs to be clear that `VERSION` is a more computer readable interpretation and `DASHBOARD_VERSION` is a more human readable one that can be used to display in the dashboard

Reply:
good comments, but also needs to apply to nuxt.config.js `version` and `dashboardVersion`